beacon/beacon.go

```diff
@@ -50,19 +49,19 @@ type Beacon interface {
 	SubmitAttestation(ctx context.Context, attestation *ethpb.Attestation, validatorIndex uint64, key *bls.PublicKey) error
 
 	// GetAggregationData returns aggregation data for the given slot and committee index
-	GetAggregationData(ctx context.Context, duty slotqueue.Duty) (*ethpb.AggregateAttestationAndProof, error)
+	GetAggregationData(ctx context.Context, duty *ethpb.DutiesResponse_Duty, key *bls.PublicKey, shareKey *bls.SecretKey) (*ethpb.AggregateAttestationAndProof, error)
```

Comment:
@nivBlox I think we should remove this for this PR

Reply:
We are not supporting aggs & propose in this merge. no ibft's for those